packages/web/package.json

```diff
@@ -35,7 +35,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint --fix src --ext .ts,.tsx"
```

Comment:
I think we can remove the `fix` flag here so we have a lint command without fixing to run on the CI.

Reply:
Ok, I will remove.